frontend/src/components/orgFacilities/FacilityList.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-lazy v-for="facility of facilities" :key="facility.facilityId" min-height="150px">
```

Comment:
Lazy loading looks great. I've tried it before and it couldn't "keep up" with fast scrolling but this is seamless.